scripts/pipeline_generator/pipeline_generator.py

```diff
@@ -53,3 +52,24 @@ def __init__(
         ):
         config.validate()
         self.config = config
+
+def read_test_steps(file_path: str) -> List[TestStep]:
+    """Read test steps from test pipeline yaml and parse them into TestStep objects."""
+    with open(file_path, "r") as f:
+        content = yaml.safe_load(f)
+    return [TestStep(**step) for step in content["steps"]]
+
+@click.command()
+@click.option("--test_path", type=str, required=True, help="Path to the test pipeline yaml file")
+@click.option("--run_all", type=str)
```

Comment:
maybe add help string that says `"1"` means true? and all other values means false?

Reply:
done